### DISCUSSION TITLE
How to customize Registration Form?

### DISCUSSION THREAD

**Discussion by abcnow** [serverpod/serverpod#920] [General]
I wonder if anybody has been able to customize/add fields to registration form from the auth module? How would you go around doing that?

**Comment by Isakdl** [comment]
I think the easiest is simply building your own custom UI in this case. You can still hook into the functionality that comes with serverpod_auth.

Take a look at this file, shipped in `serverpod_auth_email_flutter` 
https://github.com/serverpod/serverpod/blob/main/modules/serverpod_auth/serverpod_auth_email_flutter/lib/src/auth.dart

You could simply use these functions to log in, signup, reset password, etc. The UI part is of course just normal flutter stuff.
